Makefile

```diff
@@ -101,3 +103,24 @@ docs:
 
 build_spl: clean
 	python -m splunk_eventgen build --destination ./
+
```

Comment:
`make lint` will only lint the newly added and changed parts of python files here.